js/markDone.js

```diff
@@ -0,0 +1,30 @@
+import { EDIT, COMPLETE } from "/js/constants.js";
+import calculateDuration from "/js/utility/mDDuration.js";
+import createDurationElement from "/js/utility/mDDuElement.js";
+
+export default function markTodoAsCompleted(element, taskList, id) {
+  const index = taskList.findIndex((item) => item.id === id);
+  if (index === -1) {
+    return;
+  }
+  taskList[index].done = true;
+
+  const text = element.querySelector(".text").style;
+  text.textDecoration = "line-through";
+  text.color = "green";
+
+  element.querySelector(`[data-job="${COMPLETE}"]`).style.display = "none";
+  element.querySelector(`[data-job="${EDIT}"]`).style.display = "none";
```

Comment:
duplicate code should be avoided.

```suggestion
  element.querySelector(`[data-job="${COMPLETE}"]`).style.display = "none";
  element.querySelector(`[data-job="${EDIT}"]`).style.display = "none";
```

Reply:
Sorry, I couldn't understand your remark. Could you please elaborate further?

Reply:
see https://github.com/ban-wtag/ToDoAppJS/pull/4/files#r1171086150

Reply:
function overloading in real-life scenarios.!!! done!